forge/db/migrations/_20220214-02-add-templates.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Note the file has been named with a `_` at the start - this means it is ignored and not applied - so the contents can be ignored. We'll review it when we do Project Template next week.

I could have deleted it from the branch entirely, but I didn't.